playground-app/src/scenarios/share-evaluators.ts

```diff
@@ -0,0 +1,46 @@
+import type { Scenario } from "../types";
+import { contextToPrettyJson, featureDefinitionToPrettyJson } from "../utils";
+
+export const sharedEvaluator: Scenario = {
```

Comment:
I'd recommend changing the filename to match this export ("d") is missing.

Reply:
I can do that but I'm assuming you mean "s".

Reply:
`shared-evaluators.ts` I think this is the file name suggestion